packages/runtime/runtime-definitions/src/garbageCollection.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Represents the garbage collection data returned by each node in the Container. It is used for
+ * running GC in a document.
+ */
+export interface IGCData {
+    /** The GC nodes of a Fluid object in the Container. Each node has an id and a set of routes to other GC nodes. */
+    gcNodes: { [ id: string ]: string[] };
```

Comment:
This looks like a Map<string, string[]>

Reply:
I like object more than a map here simply because it works with JSON. This will be useful when writing this to summary. For map, we will have to convert to an object to use with JSON.stringify.